typescript/testsuite/execution_impl/example_testsuite_configurator.ts

```diff
@@ -0,0 +1,48 @@
+import { ExampleTestsuite } from "../testsuite_impl/example_testsuite";
+import { TestSuite } from "kurtosis-testsuite-api-lib"; //TODO (Ali)
+import * as log from "loglevel";
+import { Result, err, ok } from "neverthrow";
+import { ExampleTestsuiteArgs } from "./example_testsuite_args";
+
+
+class ExampleTestsuiteConfigurator {
+	
+	construcor () {}
+	
+	public setLogLevel(logLevelStr: string): Result<null, Error> {
+		const newLog: log.Logger = log.getLogger(logLevelStr);
+		newLog.setLevel(newLog.getLevel());
+		// log.SetFormatter(&logrus.TextFormatter{ //TOOD (Ali) - loglevel doesn't support formatting since it would stop stacktrace information
```

Comment:
Yep, that's fine - the contents of this function will be very specific to the logging framework being used